Rakefile

```diff
@@ -53,3 +54,16 @@ task :development_app do
     sh "bundle exec rails generate decidim:demo"
   end
 end
+
+desc "Generates a development app based on Docker."
+task :docker_app do
```

Comment:
I would either use `docker_development_app` for symmetry, or would just add this as an option to the `development_app`:

```
$ bundle exec rake development_app[docker]
```

See [how this can be done](http://stackoverflow.com/a/29502094)